cmd/bugzilla-backporter/main.go

```diff
@@ -131,9 +169,13 @@ func main() {
 	}
 	health := pjutil.NewHealth()
 	metrics.ExposeMetrics("ci-operator-bugzilla-backporter", prowConfig.PushGateway{}, prowflagutil.DefaultMetricsPort)
-
+	allTargetVersions, err := getAllTargetVersions(o.pluginConfig)
+	if err != nil {
+		log.WithError(err).Fatal("Error parsing plugins configuration.")
+	}
 	http.HandleFunc("/", handleWithMetrics(backporter.GetLandingHandler()))
-	http.HandleFunc("/getclones", handleWithMetrics(backporter.GetClonesHandler(bugzillaClient)))
+	http.HandleFunc("/getclones", handleWithMetrics(backporter.GetClonesHandler(bugzillaClient, allTargetVersions)))
```

Comment:
If we want this to be RESTful or RESTish let's consider

```
GET /clone/{id}
PUT|POST /clones/{id}
GET /bug/{id}
```

Reply:
for PUT|POST /clones - would the ID being in the POST body be ok?

Reply:
Yeah for create this makes sense but would hen be something like `/clones/create` as the URI

Reply:
Alright, I have made that change

Reply:
Why not POST to `/clones` ? Why  would we need a separate URI?

Reply:
So would that just be a POST to /clones - without passing an ID?